wsd/DocumentBroker.cpp

```diff
@@ -2673,23 +2673,15 @@ bool DocumentBroker::sendUnoSave(const std::shared_ptr<ClientSession>& session,
         // We do not want save to terminate editing mode if we are in edit mode now.
         //TODO: Perhaps we want to terminate if forced by the user,
         // otherwise autosave doesn't terminate?
-        oss << "\"DontTerminateEdit\":"
```

Comment:
I think it is the innocent looking changes, they introduce extra spaces that something doesn't expect. At least in the first case of unit-wopi-lock I look at

kit-3852552-3852552 2024-04-16 09:50:57.973959 +0100 [ kitbroker_002 ] ERR  ToMaster-00b: Exception while handling [save autosave= {"DontTerminateEdit" : { "type":"boolean", "value":true" },"DontSaveIfUnmodified" : { "type":"boolean", "value":true" }}]: <unspecified file>(1): expected '}' or ','| common/Session.cpp:306